pkg/analyzer/validating_webhook.go

```diff
@@ -51,8 +51,13 @@ func (ValidatingWebhookAnalyzer) Analyze(a common.Analyzer) ([]common.Result, er
 	for _, webhookConfig := range validatingWebhooks.Items {
 		for _, webhook := range webhookConfig.Webhooks {
 			var failures []common.Failure
-
+			if webhook.ClientConfig.Service == nil {
+				continue
+			}
 			svc := webhook.ClientConfig.Service
+			if svc.Name == "" || svc.Namespace == "" {
```

Comment:
@AlexsJones , I am not sure if this is necessary or if we want to actually produce an error if those attributes are blank

Reply:
ok ill remove